common/messages/messages_en.go

```diff
@@ -158,17 +151,10 @@ var messagesEn = map[string]util.Message{
 	},
 	"VolumeNotInValidState": {
 		Code:        "VolumeNotInValidState",
-		Description: "File share %s did not get valid (available) status within timeout period.",
-		Type:        util.ProvisioningFailed,
-		RC:          500,
-		Action:      "Please check your input",
-	},
-	"VolumeDeletionInProgress": {
-		Code:        "VolumeDeletionInProgress",
-		Description: "File share %s deletion in progress.",
+		Description: "File share %s did not get valid (stable) status within timeout period.",
 		Type:        util.ProvisioningFailed,
 		RC:          500,
-		Action:      "Wait for file share deletion",
+		Action:      "Please wait for some time. Run ibmcloud is share <SHARE-ID> and check the current status if is not stable then raise ticket for VPC team else on IKS team.",
```

Comment:
We need to correct this, we can discuss on this

Reply:
`Run ibmcloud is share <SHARE-ID> and check the current status if is not stable then raise ticket for VPC team else on IKS team.`